frontend/ui/network/manager.lua

```diff
@@ -246,10 +247,18 @@ function NetworkMgr:getBeforeWifiActionMenuTable()
 end
 function NetworkMgr:getSSHMenuTable()
     local SSH_port = G_reader_settings:readSetting("SSH_port") or "2222"
+    local allow_blank_password = false
     local start = function(menu)
+        local cmd = "./dropbear -E"
```

Comment:
Please use string.format instead. It's compiled so it doesn't have to be evaluated every time.

See http://wiki.luajit.org/NYI#libraries_string-library and http://wiki.luajit.org/NYI#bytecode for details.

Reply:
Oh drat! Sorry, in LuaJIT 2.1, which we use, concat *is* compiled. Hooray! :-)